src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkLambdaRuleParamsInspection.kt

```diff
@@ -43,14 +45,28 @@ class SmkLambdaRuleParamsInspection : SnakemakeInspection() {
                                 )
                             }
                             if (pyParameter.name != WILDCARDS_LAMBDA_PARAMETER) {
-                                registerProblem(
-                                        pyParameter,
-                                        SnakemakeBundle.message(
-                                                "INSP.NAME.only.these.parameters.in.section",
-                                                WILDCARDS_LAMBDA_PARAMETER,
-                                                SnakemakeNames.SECTION_INPUT
-                                        )
-                                )
+                                if (index == 0) {
+                                    registerProblem(
+                                            pyParameter,
+                                            SnakemakeBundle.message(
+                                                    "INSP.NAME.only.these.parameters.in.section",
+                                                    WILDCARDS_LAMBDA_PARAMETER,
+                                                    SnakemakeNames.SECTION_INPUT
+                                            ),
+                                            ProblemHighlightType.WARNING,
```

Comment:
Let's use weak warning or maybe remove it at all. A user could use `wd` just to make the code shorter. It is ok that we suggest `wildcards` in the completion list, but there is no warning errors if user prefers shorter name for the variable. If you decide to show weak warning please explain that according to conventions it is better to use `wildcards` name for this variable. Warning `wildcards` has to be ... - isn't correct here, nobody forbids use any name here.